hyperon_das/api.py

```diff
@@ -654,7 +647,10 @@ def query(
                 f"Invalid output format: '{extra_parameters.return_type}'"
             )
 
-        return f"{tag_not}{mapping}"
+        if query_answer.negation:
+            return {'tag_not': False, 'mapping': result}
```

Comment:
(1) The boolean is inverted here. "tag_not" should mean the result is supposed to be negated, thus "tag_not" should be True rather than False.
(2) The name "tag_not" is awkward. My suggestion is "negation". I suppose you don't like it since you changed it :-). If that's the case make other suggestions.
(3) Why are you wrapping only answers whose "negation" is True? If you are wrapping around a dict  with an explicit flag to indicate negation, you should wrap ALL answers. (non-negation answers should have the flag set to False).

Reply:
1) Sorry, my bad. I got confused, I thought True and put False. I'll change this
2) Everything is fine. 'negation' name is better. Sorry, I'm really bad with names :(.
3) I thought about this, wrapping all the answers, but most use cases (at this point) don't return this flag, right? So I thought that for users a dict might not be that useful. But if it's not a problem for you to always return a dict that I can change

Reply:
Regarding (3). It's not a matter of preference, I think returning different types of objects depending on the query answer is confusing. We can re-discuss this API but for now I prefer having a homogeneous return type.